barretenberg/cpp/src/barretenberg/commitment_schemes/gemini/gemini.test.cpp

```diff
@@ -10,15 +11,16 @@ template <class Curve> class GeminiTest : public CommitmentTest<Curve> {
     using GeminiProver = GeminiProver_<Curve>;
     using GeminiVerifier = GeminiVerifier_<Curve>;
     using Fr = typename Curve::ScalarField;
-    using GroupElement = typename Curve::Element;
+    using Commitment = typename Curve::AffineElement;
 
   public:
     void execute_gemini_and_verify_claims(std::vector<Fr>& multilinear_evaluation_point,
-                                          std::vector<Fr>& multilinear_evaluations,
+                                          RefVector<Fr> multilinear_evaluations_unshifted,
+                                          RefVector<Fr> multilinear_evaluations_shifted,
                                           RefSpan<Polynomial<Fr>> multilinear_polynomials,
                                           RefSpan<Polynomial<Fr>> multilinear_polynomials_to_be_shifted,
-                                          RefVector<GroupElement> multilinear_commitments,
-                                          RefVector<GroupElement> multilinear_commitments_to_be_shifted)
+                                          RefVector<Commitment> multilinear_commitments,
```

Comment:
I unified the use of GroupElement of Commitment in the commitment_schemes folder as the fact that half of the code was using GroupElement and half Commitment in function signatures was creating annoying compiling error when couppling protocol. Now we only use GroupElement explicitly for operations that require it. As I understand, this is fine because the only place where we need to be careful is when adding stuff to transcript (we shouldnt add projective elements to transcript.)